src/Tribe/Aggregator/Record/Queue_Realtime.php

```diff
@@ -97,7 +97,7 @@ public function render_update_message() {
 			<li class="tracked-item track-skipped"><strong><?php esc_html_e( 'Skipped:', 'the-events-calendar' ); ?></strong> <span class="value"></span></li>
 		</ul>
 		<div class="progress-container">
-			<div class="progress" title="<?php echo esc_html( sprintf( __( '%d%% complete', 'the-events-calendar' ), $percent ) ); ?>">
+			<div class="progress" title="<?php echo esc_html( sprintf( _x( '%d%% complete', 'The "percent complete" of a process, e.g. "45% complete"', 'the-events-calendar' ), $percent ) ); ?>">
```

Comment:
`esc_html_x()`

Reply:
Not needed here because it's wrapped in an `esc_html()` function.